sections/docker.zsh

```diff
@@ -24,8 +24,29 @@ spaceship_docker() {
 
   spaceship::exists docker || return
 
+  # Better support for docker environment vars: https://docs.docker.com/compose/reference/envvars/
+  local compose_exists=false
+  if [[ -n "$COMPOSE_FILE" ]]; then
+    # Use COMPOSE_PATH_SEPARATOR or colon as default
+    local separator="${COMPOSE_PATH_SEPARATOR=":"}"
```

Comment:
Wouldn't `${COMPOSE_PATH_SEPARATOR:-":"}` be preferable to `${COMPOSE_PATH_SEPARATOR=":"}` here to avoid setting `COMPOSE_PATH_SEPARATOR` as a side-effect to the assignment to `separator`?

Reply:
Of course, that was my lack of zsh syntax knowledge! I had to google it! ;)

Done

PS: is there some way to write a some kind of test?

Reply:
Sorry @juracy, There are no tests in spaceship-prompt now. But we're looking forward to it with #75